cmatrix.c

```diff
@@ -598,7 +599,14 @@ if (console) {
                     bold = 2;
                     break;
                 case 'L':
+                    if (lock == 1) {
+                        lock_count++;
+                    }
                     lock = 1;
```

Comment:
Why ? This causes the bug where you press four times L and it breaks. See previous comment.